blint/cli.py

```diff
@@ -86,6 +86,20 @@ def build_parser():
         dest="suggest_fuzzable",
         help="Suggest functions and symbols for fuzzing based on a dictionary.",
     )
+    # TODO: what to do should this be default?
+    parser.add_argument(
```

Comment:
@aryan-rajoria We shouldn't make this the default unless it's applicable more often than not.